cypress/support/pages/settings.pageObject.js

```diff
@@ -0,0 +1,59 @@
+import PageObject from '../PageObject';
+
+class SettingsPageObject extends PageObject {
+  url = '/settings';
+
+  get usernameField() {
+    return cy.getByDataCy('username_field');
+  }
+
+  get bioField() {
+    return cy.getByDataCy('bio_field');
+  }
+
+  get emailField() {
+    return cy.getByDataCy('email_field');
+  }
+
+  get passwordField() {
+    return cy.getByDataCy('password_field');
+  }
+
+  get updateBtn() {
+    return cy.getByDataCy('update_btn');
+  }
+
+  typeUsername(username) {
+    this.usernameField.clear().type(username);
+  }
+
+  isUpdatedBio(value) {
+    this.bioField.should('have.value', value);
+  }
+
+  isUpdEmail(value) {
+    this.emailField.type(value);
+  }
+
+  isUpdPassword(value) {
```

Comment:
do not understand how do you assert the changed password here? you just type new password. So change the name of method to fillThePasswordField()